dev/row_extractor.h

```diff
@@ -76,12 +149,12 @@ namespace sqlite_orm {
      */
     template<class V>
     struct row_extractor<V, std::enable_if_t<std::is_arithmetic<V>::value>> {
-        V extract(const char* row_value) const {
-            return this->extract(row_value, tag());
+        V extract(const char* colTxt) const {
```

Comment:
I agree, and that's why I wrote out the variable names in the primary template and shortened them in the others, to keep them - well - short, because they just repeat. Am not quarrelsome though :)